documentation/adoc/metrics.adoc

```diff
@@ -1,149 +1,137 @@
-# Metrics
+== Metrics
 
 This section describes how it's possible to deploy a Prometheus server for scraping metrics from the Kafka cluster and showing them using a Grafana dashboard. The provided resources for deploying can't be considered production grade; they are used just as an example to show how it's possible to use the metrics feature provided in this project.
 
 If you are going to use `minikube` or `minishift` for your Apache Kafka deployment adding Prometheus and Grafana servers, it would be better to increase the default amount of memory available to the virtual machine (i.e. not using the default 2 GB of RAM but 4 GB instead).
```

Comment:
The whole thing about memory is already described in the guide for installing Minikube / Minishift. HEre I would probably just mention that at least 4GB of memory might be needed and link to the minikube / minishift section about how to increase the memory.

Reply:
Right! I did not pay attention to the Appendix B before.